Packs/GuardiCore/Integrations/integration-GuardiCore.yml

```diff
@@ -772,14 +796,19 @@ script:
 
     delete(args['raw-response']);
 
+    var res_login =  sendRequest(methodDictionary.login, urlDictionary.login, undefined, {username: params.credentials.identifier, password: params.credentials.password})
+
+    if(JSON.stringify(res_login) == '{}'){
+        return 'Authentication at login failed '
+    }
```

Comment:
please use the standard indentation of JS.
also, why re you returning and not throwing an error?

Reply:
Because maybe it can be an empty response? status code 200 and empty body happens on logout, I add it to handle cases such these. (logout request is now not here but on different function)